frontend/src/main/java/com/google/tripmeout/frontend/PlaceVisitModel.java

```diff
@@ -11,11 +14,11 @@ public static Builder builder() {
   }
 
   public abstract String placeId();
-  public abstract String name();
+  @Nullable public abstract String name();
   public abstract String tripId();
   public abstract UserMark userMark();
-  public abstract double latitude();
-  public abstract double longitude();
+  @Nullable public abstract Double latitude();
```

Comment:
Just going on the record, `@Nullable` on a `Double` field is usually a bad idea. At some point you're going to rely on auto unboxing accidentally and get a runtime error you're not expecting. But I really don't care for now on this pr.